fuse_core/include/fuse_core/eigen.h

```diff
@@ -66,6 +68,14 @@ using Matrix9d = Eigen::Matrix<double, 9, 9, Eigen::RowMajor>;
 template <typename Scalar, int RowsAtCompileTime, int ColsAtCompileTime>
 using Matrix = Eigen::Matrix<Scalar, RowsAtCompileTime, ColsAtCompileTime, Eigen::RowMajor>;
 
+template<typename Derived>
+std::string to_string(const Eigen::DenseBase<Derived>& m)
+{
+  std::ostringstream oss;
+  oss << m;
```

Comment:
Do you want to do any fancy Eigen formatting to make it look prettier?
E.g.
```
Eigen::IOFormat pretty(4, 0, ", ", "\n", "[", "]");
oss << m.format(pretty) << "\n"
```
Or whatever you preferred format might be?

Reply:
Sure thing. Done

Reply:
Sure thing. Done